src/main/resources/io/jenkins/plugins/designlibrary/AppBar/index_fr.properties

```
@@ -0,0 +1,2 @@
+title=Titre de la page
+description=Les barres d'applications fournissent l'en-tête de la page ainsi que des actions importantes à propos de la page actuelle. Elles doivent être incluses avant le <code>l\:main-panel</code>.
```

I suggest the following change
```suggestion
description=Les barres d'applications fournissent l'en-tête de la page ainsi que des actions importantes à propos de la page actuelle. Elles doivent être incluses avant le <code>l:main-panel</code>.
```